.resinci.yml

```diff
@@ -1,3 +1,19 @@
 ---
 docker:
   publish: false
+npm:
+  platforms:
+    - name: linux
+      os: ubuntu
+      architecture: x86_64
+      node_versions:
+        - "10"
+        - "12"
+        - "14"
+    - name: linux
+      os: alpine
+      architecture: x86_64
+      node_versions:
+        - "10"
+        - "12"
+        - "14"
```

Comment:
> CI: enable CI for the same platforms as balena-cli

That's fair, but... Considering that the `balena-preload` tests are currently limited to `npm run lint`, we could actually have a `.resinci.yml` more like [the one from balena-multibuild](https://github.com/balena-io-modules/balena-multibuild/blob/v4.11.0/.resinci.yml):
```
npm:
  platforms: []
```
Alternatively, we could add `catch-uncommitted` to the `test` npm script -- see other comment -- so that at least there is a chance that the build could fail. Would probably not need to test it in alpine, ubuntu and 3 different versions of node though. 😁

Reply:
I tried to enable only alpine node 10 but I guess ubuntu 10 & 12 are required by the project so I can stick to those for now.